ray/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM jupyter/pyspark-notebook
+
+MAINTAINER Alexey Tumanov <atumanov@gmail.com>
+
+RUN pip install ray && \
```

Comment:
Good idea, the Ray version will change between now and the camp, but we could use `tensorflow==1.3.0` and `gym==0.9.2`.